src/metabase/models/params/chain_filter.clj

```diff
@@ -115,26 +115,21 @@
 
 (defn- filter-clause
```

Comment:
I think we should use the `Constraint` schema fro this function as well.

Reply:
done in 39ec5a256ca0cd926f3cbad9232029374e1191fe